include/util/exception.hpp

```diff
@@ -1,5 +1,5 @@
-#ifndef OSRM_EXCEPTION_HPP
-#define OSRM_EXCEPTION_HPP
+#ifndef EXCEPTION_HPP
+#define EXCEPTION_HPP
```

Comment:
I deem these common header guard names dangerous if they do not at least include some directory/other identifier. Good chance of header guard conflicts.